README.md

```diff
@@ -0,0 +1,15 @@
+Welcome to Tic Tac Toe!
+
+About:
+   - This Tic Tac Toe game was created to help me learn C#. The tests are written using [xUnit](https://xunit.net/). At this time, it can be run in the console. 
+
+Setup:
+- [Install dotnet](https://dotnet.microsoft.com/en-us/download). This project uses version 5.0.
```

Comment:
nit: just curious, why not 6.0, which seems to be the latest according to https://docs.microsoft.com/en-us/dotnet/core/install/macos ?

Reply:
No worries, I was just curious. Turns out `brew install dotnet` also pulls in a 5.0 version, so this may be more convenient to use anyway, at least until you need a 6.0 feature.